index.html

```diff
@@ -120,12 +120,12 @@
         window.addEventListener(
             'error', loadErrHandler, true /* capture phase type of listener */);
     </script>
-    <script><!--#include virtual="/config.js" --></script><!-- adapt to your needs, i.e. set hosts and bosh path -->
```

Comment:
@guusdk why do you need this change? Can't you handle this in your webserver's configs?